.travis.yml

```diff
@@ -41,6 +41,8 @@ script:
     - cd $TRAVIS_BUILD_DIR/dist/Orange-*
     - python -c "from Orange.tests import *"
     - PYTHONUNBUFFERED=x coverage run --source=Orange setup.py test
+    - cd $TRAVIS_BUILD_DIR
+    - ./bin/check_pylint_diff
```

Comment:
Can you make it only run once, e.g., in Python 3.4 build, or better still in a separate build task that can fail fast? I believe you can do so by [setting an env variable in the build matrix](https://docs.travis-ci.com/user/customizing-the-build/#Explicity-Including-Builds) and then only conditionally linting if the variable is set.